liquidctl/cli.py

```diff
@@ -9,6 +9,7 @@
   liquidctl [options] set <channel> color <mode> [<color>] ...
   liquidctl --help
   liquidctl --version
+  liquidctl [-f] --validate
```

Comment:
Does this work?  I think it would need to be `liquidctl [-f <file>] --validate` for docopt to parse it correctly.  Additionally, `--validate` needs to be listed as an option, or be a command.

Finally, at this point, it may be unclear to the user what `--validate` validates... maybe `--validate-config` or `--check-config`?  (This part may no longer apply if `--file` is renamed to `--config`).

Reply:
> Does this work? I think it would need to be liquidctl [-f <file>] --validate for docopt to parse it correctly. Additionally, --validate needs to be listed as an option, or be a command.

I did test this and I thought it worked as expected, but Ill check again. 


> Finally, at this point, it may be unclear to the user what --validate validates..

I know theat nginx using a `-t` flag to do that validation and they call it test. perhaps that would be better. And I do like the `--config` flag instead of `--file` but I also like short flags more than long names and I dont think that `-c` would make sense.